q2_taxa/plugin_setup.py

```diff
@@ -35,6 +35,19 @@
     },
     parameters={'level': qiime2.plugin.Int},
     outputs=[('collapsed_table', FeatureTable[Frequency])],
+    input_descriptions={
+        'taxonomy': 'Taxonomic annotations for all features.',
```

Comment:
Added more details:

*Taxonomic annotations for features in the provided feature table. All features in the feature table must have a corresponding taxonomic annotation. Taxonomic annotations that are not present in the feature table will be ignored.*